README.md

```diff
@@ -18,24 +18,25 @@ This repo contains a [template `local_nb_nodes.json`](/local_nb_nodes.json) file
 
 Examples:  
 
-`local_nb_nodes.json` with one local node
+`local_nb_nodes.json` with one local node API running on `http://localhost:8000`
 ```json
 {
-    "NodeName": "First Node",
-    "ApiURL": "https://firstnode.org"
+    "NodeName": "Local graph",
+    "ApiURL": "http://host.docker.internal:8000"
 }
 ```
+_**NOTE:** If the local node API(s) you are federating over is running on the same host machine as the federation API (e.g., the URL to access the node API is `http://localhost:XXXX`), in `local_nb_nodes.json` you **must** replace `localhost` with `host.docker.internal` in the corresponding `"ApiURL"`, as shown above._
 
 `local_nb_nodes.json` with two local nodes
 ```json
 [
     {
-        "NodeName": "First Node",
-        "ApiURL": "https://firstnode.org"
+        "NodeName": "Local graph 1",
+        "ApiURL": "http://host.docker.internal:8000"
     },
     {
-        "NodeName": "Second Node",
-        "ApiURL": "https://secondnode.org"
+        "NodeName": "Local graph 2",
+        "ApiURL": "http://192.168.0.1"
```

Comment:
any reason you change from name to IP here? Both work, just curious

Reply:
To try to stay more consistent with the `"ApiURL"`s from the examples in our docs!

https://neurobagel.org/federate/#local_nb_nodesjson-configuration-file